src/utils/parseIntents.ts

```diff
@@ -53,7 +54,11 @@ function parseIntent(
         action: node.props.action
           ? node.props.action.startsWith('http')
             ? node.props.action
-            : parsePath(options.baseUrl + node.props.action) +
+            : parsePath(
+                (node.props.action.startsWith('~')
+                  ? options.initialBaseUrl
+                  : options.baseUrl) + node.props.action.slice(1),
```

Comment:
in order to drop delete the `'~'` root path symbol prefix

Reply:
Can we not do this for the `baseUrl` case?

Reply:
Sorry a bit confused here, wdym?

If you're referring to the ternary – if the ~ prefix was found it sets the initial base url from the very top-level frog instance, otherwise sets the base url from the current frog instance.

That prefix later is sliced to get the correct url.

Can you elaborate in a greater detail what not to do?

Thanks

Reply:
I mean, if it does not start with `~` (latter ternary case), then we don’t want to slice right?